src/ServiceControlInstaller.Engine/Instances/MonitoringTransports.cs

```diff
@@ -8,50 +8,102 @@ public class MonitoringTransports
+            //      As a result the Match predicate should evalute to true both for TypeName and human friendly name.
+            //      Matching separately on Name and TypeName would not be enough because we need to be backwards compatible.
+            //      As a result Match is comparing to old/current names, old/current types.
             new TransportInfo
             {
-                Name = "AzureServiceBus",
+                Name = TransportNames.AmazonSQS,
+                ZipName = "AmazonSQS",
+                TypeName = "ServiceControl.Transports.AmazonSQS.ServiceControlSqsTransport, ServiceControl.Transports.AmazonSQS",
+                SampleConnectionString = "AccessKeyId=<ACCESSKEYID>;SecretAccessKey=<SECRETACCESSKEY>;Region=<REGION>;QueueNamePrefix=<prefix>",
+                Help = "'AccessKeyId', 'SecretAccessKey' and 'Region' are mandatory configurations.",
+                Matches = name => name.Equals(TransportNames.AmazonSQS, StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("ServiceControl.Transports.AmazonSQS.ServiceControlSqsTransport, ServiceControl.Transports.AmazonSQS", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("NServiceBus.SqsTransport, NServiceBus.AmazonSQS", StringComparison.OrdinalIgnoreCase)
+            },
+            new TransportInfo
+            {
+                Name = TransportNames.AzureServiceBusEndpointOrientedTopology,
+                ZipName = "LegacyAzureServiceBus",
+                TypeName = "ServiceControl.Transports.LegacyAzureServiceBus.EndpointOrientedTopologyAzureServiceBusTransport, ServiceControl.Transports.LegacyAzureServiceBus",
+                SampleConnectionString = "Endpoint=sb://[namespace].servicebus.windows.net; SharedSecretIssuer=<owner>;SharedSecretValue=<someSecret>",
+                Matches = name => name.Equals(TransportNames.AzureServiceBusEndpointOrientedTopology, StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("ServiceControl.Transports.LegacyAzureServiceBus.EndpointOrientedTopologyAzureServiceBusTransport, ServiceControl.Transports.LegacyAzureServiceBus", StringComparison.OrdinalIgnoreCase)
+            },
+            new TransportInfo
+            {
+                Name = TransportNames.AzureServiceBusForwardingTopology,
+                ZipName = "LegacyAzureServiceBus",
+                TypeName = "ServiceControl.Transports.LegacyAzureServiceBus.ForwardingTopologyAzureServiceBusTransport, ServiceControl.Transports.LegacyAzureServiceBus",
+                SampleConnectionString = "Endpoint=sb://[namespace].servicebus.windows.net; SharedSecretIssuer=<owner>;SharedSecretValue=<someSecret>",
+                Matches = name => name.Equals(TransportNames.AzureServiceBusForwardingTopology, StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("AzureServiceBus", StringComparison.OrdinalIgnoreCase)
```

Comment:
Is there any issue with this match being the same as the new entry below for the new transport?

Reply:
The new entry is "Azure Service Bus" and the old one is "AzureServiceBus"

Reply:
Cool, wasn't sure if there was a problem or not, just wanted to check.